### DISCUSSION TITLE
利用深度學習評估二氧化碳最佳儲存地點

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#175] [General]
近期許多科學家盡力於解決（全球暖化）溫室氣體排放過度的問題。其中將二氧化碳儲藏到地底下可助於對抗大氣變暖，但找到可以安全用作儲藏庫的合適地點相當複雜且不易尋找且評估。例如：如果地底下的壓力超過一定限度，就會土壤或地層將出現裂縫注入的二氧化體仍會洩漏光; 如果注入過多的氣體，亦會出現類似的問題。

為了解決這複雜的問題，賓夕法尼亞州立大學領導的研究團隊將人工智慧技術與對物理學理論結合，開發出一種高效、經濟的預測建模方法。數值模擬和複雜詳細的模型可用於幫助理解人們無法短時間內解決的問題。其中預測潛在地點對二氧化碳注入的反應為近期的應用之一。然而，這些模擬運行起來可能非常昂貴且耗時，對於每個被探索為候選存儲地點的新地點，必須重新運行新的數值模擬。

為避免數值模擬所需消耗過多的成本和時間，研究團隊訓練且致力於研究更高階的深度學習算法，以便在各種場景中做出準確的預測。從 7,500 英尺深的儲存庫中二氧化碳模擬情景產生的數據中學習，這些算法能夠預測二氧化碳飽和度和壓力在新模擬系統中的表現。

該團隊提出的演算法建模框架顯示於下圖。（該網站只公布前半的建模框架）

<img width="448" alt="Screen Shot 2021-12-01 at 13 30 24" src="https://user-images.githubusercontent.com/91121500/144177464-964d350a-f3ef-470a-8525-ea3453de9fc3.png">

該研究解果可以為該領域的科學家或操作員提供可靠的預測軟體。各個使用者可以選擇不同的注入選項，並根據所使用的機器在幾秒鐘內查看二氧化碳行為的預測，以達到快速尋找儲存地，解決二氧化碳排放問題。

資料來源：https://www.psu.edu/news/engineering/story/physics-informed-deep-learning-assess-carbon-dioxide-storage-sites/

**Comment by EricAgumonJou** [comment]
<img width="565" alt="Screen Shot 2021-12-01 at 13 32 42" src="https://user-images.githubusercontent.com/91121500/144177908-717dbddd-3ccb-4503-af4e-37aecca06a0f.png">
該圖為碳捕獲和封存地關係圖，其有助於減少大氣中的二氧化碳排放（賓夕法尼亞州立大學領導的研究團隊開發了一種建模技術，可以更輕鬆、更便宜地識別存儲地點）。

**Comment by EricAgumonJou** [comment]
模擬訓練數據近似於系統的性能，算法從中識別出可用於估計未來行為的pattern——但這些pattern並不總是遵守物理定律。
儘管該模型能有效地訓練被訓練，但由於多種原因（包括數據不準確），該模型可能會對系統做出不正確的預測。有限的訓練數據可能會導致估計過於針對數據集量身定制，這種問題稱為過度擬合（overfitting）。

研究人員通過結合物理定律來改進深度學習算法的預測，開發受基本物理學原理（例如：地下液體的自然運動或質量守恆定律...等）約束模型，從而解決了此一缺點。當出現基於物理的差異時，該團隊會添加補償（penalty），以幫助演算法學習糾正錯誤。

賓州大學的教授 Shokouhi 說，這種方法產生的模型仍然比傳統的數值模擬更便宜、使用速度更快，但又比數據驅動模型更準確，且可能更具有更加的普及性。

**Comment by HSU-YU-JEN** [comment]
沒想到現在也可以透過AI學習情境，分析適合儲存碳的地方，如果技術成熟，相同的技術應該也能被用於地質探勘等地方

**Comment by MAKERRRofKEVIN** [comment]
國內目前有在進行相關研究的是成大的資源工程系，實作的地點是苗栗的出磺坑油田，不過是使用比較傳統的探測技術，或許未來也能夠有機會看到此技術在台灣的應用

**Comment by 107602505** [comment]
目前二氧化碳封存的議題在台灣也蠻夯的，沒想到深度學習也能應用在這上面，但目前看起來它的不確定性還是蠻高的，希望未來能成功發展出來。

**Comment by zixunn** [comment]
希望將來這項技術能順利問世，並運用在二氧化碳重點排放國等，成為對付全球暖化的武器。

**Comment by syt-110** [comment]
感覺很多關於二氧化碳及氣候變遷模擬與議題逐漸被重視
